src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentService.kt

```diff
@@ -43,13 +49,28 @@ class IncidentService(
       incidentDateTime = LocalDateTime.of(this.incidentDate, this.incidentTime),
       reportedStaff = this.reportingStaff.toStaff(),
       reportedDateTime = LocalDateTime.of(this.reportedDate, this.reportedTime),
-    )
-
-  fun uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Staff.toStaff() =
-    Staff(
-      staffId = id,
-      firstName = firstName,
-      lastName = lastName,
-      username = accounts.maxByOrNull { it.type }?.username ?: "unknown",
+      staffParties = this.staffParties().map { it.staffParty().toStaff() },
+      offenderParties = this.offenderParties().map { it.offenderParty().toOffender() },
     )
 }
+
+private fun Incident.staffParties(): List<IncidentParty> =
+  this.parties.filter { it.isStaffParty() }
+
+private fun uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Staff.toStaff() =
```

Comment:
Can you create an import alias for this instead of full package?

Reply:
I noticed that too - yep, a better solution